index.js

```diff
@@ -0,0 +1,36 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const express = require('express');
+const session = require('express-session');
+
+const app = express();
+const router = require('./app/router');
+const middlewares = require('./middlewares/middlewares');
+
+app.set('view engine', 'ejs');
+app.set('views', './app/views');
+
+app.use(express.static('public'));
+
+const port = 8080;
```

Comment:
```suggestion
const port = process.env.PORT || 8080;
```

et bouge cette ligne proche de son utilisation (ie, `app.listen`)